app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "decrediton",
   "productName": "decrediton",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Electron application boilerplate based on React, React Router, Webpack, React Hot Loader for rapid application development",
```

Comment:
Done.  No idea how I missed that one when it was right under something i did edit.